functional-samples/tutorial.reading-time/README.md

```diff
@@ -0,0 +1,14 @@
+# Tutorial: Run scripts on every page
+
+This sample demonstrates how to _run scripts on every page_ using an extension called _Reading Time_.
```

Comment:
In practice, we only support certain documentation pages on developer.chrome.com. Could you update this introduction and the "Running this extension" section to reflect that?

Reply:
Thanks for catching that! Updated the wording to be more specific.